### ISSUE TITLE
Username-password combinations is not working after a brand-new installation

### ISSUE THREAD

**Issue by valentierra** [tiangolo/full-stack-flask-couchbase#2]
I am running my local dev-environment on a macOS and the docker swarm on VirtualBox Ubuntu servers on the same machine.

I have already set up the DNS subdomains and add dev.<mydomain> in the /etc/hosts file related with 0.0.0.0 IP address.

Also checked out the .yml files to verify the SERVER_NAME issue reported before is not affecting full-stack couchbase version.

The problem seems to have affected the following addresses: localhost, localhost/api, http://localhost/swagger/, and localhost:8091 (Couchbase)

I've also tried http://localhost.tiangolo.com and dev.<mydomain> with identical results. I will study the logs in further detail to see where the problem is originated

**Comment by tiangolo** [comment]
I see. 

In that case, your VirtualBox machine will have a specific IP address, it will be different than `127.0.0.1`, which is the one for `localhost`, which is also used under the hood by Docker-for-Mac's VM (virtual machine). 

If you set it up with Docker Toolbox (which uses VirtualBox), the default IP address will be `192.168.99.100`. If that's the case, you can use http://dockertoolbox.tiangolo.com or any subdomain of that.

If the IP address of your VirtualBox VM is different, let's say it's `192.168.1.5`, then you have to update your `/etc/hosts` file and put a section with your VM's IP, e.g.:

```
192.168.1.5    dev.my-super-app.com
```

**Comment by valentierra** [comment]
Thanks for the advice Tiangolo!

Problem solved! It was a 100% a fault from my part.

It was caused by a sync feature of my browser. I recently changed to Vivaldi, which BTW is quite fast and configurable, and used the same Vivaldi user with dev sessions on different machines. That caused a mess with authorization tokens, despite of having made a brand-new installation of the Full-Stack-Flask-Couchbase.

The moral of the fable:
1 - Don't use browser sync features from different dev machines sessions.
2 - Have just one development environment for each purpose.

And so, I am using Full-Stack only through a development VirtualBox machine running Ubuntu and leave the macOS to deal with other development tools and environments. 

It was as simple as added the address 0.0.0.0 related to dev.mydomain.com into the Ubuntu's dev machine _/etc/hosts_ file. 

To figure out which IPs are exposed by each VM (dev and server machines) I used the **ifconfig | grep "inet"** command and then connected each VM by ssh to be sure which IP works. Knowing the IPs, now is time to configure the docker swarm.

I figure I can connect my dev. machine from the macOS browser by exposing Ubuntu's dev machine IP and adding that IP to the macOS etc/hosts file. As both are in the same Mac, It is not necessary but it can be fun to know it is possible that way.
